Tools/environment_install/install-prereqs-ubuntu.sh

```diff
@@ -24,7 +30,13 @@ fi
 
 # GNU Tools for ARM Embedded Processors
 # (see https://launchpad.net/gcc-arm-embedded/)
-ARM_ROOT="gcc-arm-none-eabi-6-2017-q2-update"
+if $IS_WINDOWS_10_WSL; then
+	ARM_ROOT="gcc-arm-none-eabi"
```

Comment:
This gives a 404 error when the script attempts to download the arm compiler

Reply:
doh!

Reply:
do you have a better suggestion?

Reply:
gcc-arm-none-eabi-6-2017-q2-update-win32-sha2.exe ?